src/OpenTelemetry.Abstractions/Trace/Config/TraceParams.cs

```diff
@@ -26,24 +26,24 @@ public sealed class TraceParams : ITraceParams
         /// Default trace parameters.
         /// </summary>
         public static readonly ITraceParams Default =
-            new TraceParams(Samplers.GetProbabilitySampler(DefaultProbability), DefaultSpanMaxNumAttributes, DefaultSpanMaxNumAnnotations, DefaultSpanMaxNumMessageEvents, DefaultSpanMaxNumLinks);
+            new TraceParams(Samplers.GetProbabilitySampler(DefaultProbability), DefaultSpanMaxNumAttributes, DefaultSpanMaxNumEvents, DefaultSpanMaxNumMessageEvents, DefaultSpanMaxNumLinks);
 
         private const double DefaultProbability = 1e-4;
         private const int DefaultSpanMaxNumAttributes = 32;
-        private const int DefaultSpanMaxNumAnnotations = 32;
+        private const int DefaultSpanMaxNumEvents = 32;
         private const int DefaultSpanMaxNumMessageEvents = 128;
```

Comment:
if you are removing message events - this parameter (`DefaultSpanMaxNumMessageEvents `) doesn't make sense any longer

Reply:
Plan to address this in a separate PR for #16